### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Urs Behrmann, Rodrigo Lopes dos santos

### DISCUSSION THREAD

**Discussion by RodrigoLopesDos** [heig-vd-dai-course/heig-vd-dai-course#493] [Show and tell]
Un chat multi-users dans le quel il est possible d'échanger des messages texte.

1. Connexion chat -> Choix user name si celui-ci est disponible alors on accède au chat sinon on doit en trouver un disponible.
2. Dans l'application on aura :
- liste des users (indicateurs si connecté)
- liste de conversations (dans une conversation ->1 user à n users)
- Il est possible de rejoindre, quitter, créer une conversation
- message non permanant

Optionnel:
- message permanant sur maximum 24h

**Comment by GeraudSilvestri** [comment]
C'est bon pour nous

**Comment by UBehrmann** [comment]
Link to repo: https://github.com/UBehrmann/DAI_practical_work_2
link to the release commit: https://github.com/UBehrmann/DAI_practical_work_2/commit/75aa9fe5d6a507cfc9522d01e2675d21e58de0c6
